omnibus/config/software/python3.rb

```diff
@@ -1,6 +1,6 @@
 name "python3"
 
-default_version "3.7.4"
+default_version "3.8.0"
```

Comment:
Would it make sense to use [3.8.1](https://www.python.org/downloads/release/python-381/) instead?

Reply:
Absolutely! It was not available when this PR has been started but I have to update this! Thank you.

Reply:
I thought so! 
Sorry for being a PITA with this - you might need to update omnibus python checksum too 🙏

Reply:
Oh nvm my oversight, that's pip, not python.

Reply:
You were right it was needed and don't worry, not a PITA, quite the contrary!